django_freeradius/api/views.py

```diff
@@ -316,3 +316,32 @@ def post(self, request, *args, **kwargs):
             return Response(response)
 
     obtain_auth_token = csrf_exempt(ObtainAuthTokenView.as_view())
+
+    class ValidateAuthTokenView(generics.CreateAPIView):
+        radius_token = RadiusToken
+
+        def get_or_create_radius_token(self, user):
+            """
+            Designed to be overridden by extensions
+            """
+            radius_token, rad_token_created = self.radius_token.objects.get_or_create(user=user)
+            return radius_token
```

Comment:
please avoid repeating the code of this method which is copied from `ObtainAuthTokenView` by creating a mixin which is added to both views